backend/src/libs/packages/server-application/base-server-application-api.ts

```diff
@@ -0,0 +1,58 @@
+import swaggerJsdoc from 'swagger-jsdoc';
+
+import { AppEnvironment } from '#libs/enums/enums.js';
+import { type Config } from '#libs/packages/config/config.js';
+
+import { type ServerApplicationApi } from './libs/interfaces/interfaces.js';
+import { type ServerApplicationRouteParameters } from './libs/types/types.js';
+
+class BaseServerApplicationApi implements ServerApplicationApi {
+  public version: string;
+
+  public routes: ServerApplicationRouteParameters[];
+
+  private config: Config;
+
+  public constructor(
+    version: string,
+    config: Config,
+    ...handlers: ServerApplicationRouteParameters[]
+  ) {
+    this.version = version;
+    this.config = config;
+    this.routes = handlers.map((item) => ({
```

Comment:
```suggestion
    this.routes = handlers.map((handler) => ({
```
Maybe naming this variable something like 'handler' would be clearer
'item' is used in multiple files, same for others as well

Reply:
Changed